xvk.hpp

```diff
@@ -42,10 +42,11 @@ namespace xvk {
 			}
 		}
 		
-		Instance(const Instance&) = delete;
-		Instance& operator=(const Instance&) = delete;
-		Instance(Instance&&) = delete;
-		Instance& operator=(Instance&&) = delete;
+		// NOT always can refer as pointers
```

Comment:
This deletes the copy/move constructors... 
Why is this an issue ?

Reply:
I getting error when located insider in class with copy operation (even if wrapped)… for example, my `vkt::uni_ptr` doesn't work when used reference. I don’t know for sure, maybe this is my mistake. I will try to find a solution on my side as well.

Reply:
This object should not be copied because its destructor destroys the vulkan instance. 

Also, you do not need to have smart pointers around the Device and Instance objects, you can use raw pointers instead, but if you really want to use smart pointers you need to construct it in place passing the constructor arguments directly into the smart pointer creator method so that you do not copy the entire object. 

Example using std smart pointers :
```
std::unique_ptr<xvk::Instance> instance = std::make_unique<xvk::Instance>(&loader, &createInfo);
```

Reply:
I seem to have found a solution to my problem... basically it concerns `const T &` from `const` versions of `vkt::uni_ptr`...

Reply:
Great, so is it working corectly now with the original version (or latest master branch) with your use case ?